lib/src/main/kotlin/com/lemonappdev/konsist/core/provider/KoReturnStatementsProviderCore.kt

```diff
@@ -1,15 +1,15 @@
 package com.lemonappdev.konsist.core.provider
 
 import com.lemonappdev.konsist.api.provider.KoReturnStatementsProvider
-import org.jetbrains.kotlin.psi.KtFunction
+import org.jetbrains.kotlin.psi.KtElement
 
 internal interface KoReturnStatementsProviderCore : KoReturnStatementsProvider {
-    val ktFunction: KtFunction
+    val ktElement: KtElement
 
     override val numReturnStatements: Int
         get() {
             val regex = Regex("return")
```

Comment:
This should be `"return "` , otherwise we will have a false positive e.g.
```kotlin
val sampleProperty: Int
    get() {
        val returnValue = "aaa"
        return returnValue
    }

```
```